File: tests/fixtures/go/clean_restate.go
package main

func RetryUpload(id int) {
	// retry because the upstream service returns 503 sometimes
	upload(id)
}

func CacheManifest(cache map[string]string, raw string) {
	// memoizes the parsed manifest for repeated lookups
	cache["manifest"] = parse(raw)
}

// Increment increases the shared counter by one.
func Increment(counter int) int {
	return counter + 1
}

func RefreshToken(token string) string {
	// nolint: keep the legacy token format for older clients
	legacy := fetchLegacyToken(token)
	return legacy
}

File: tests/fixtures/python/clean_restate.py
def retry_upload(item_id):
    # retry because the upstream service returns 503 sometimes
    upload(item_id)


def cache_manifest(cache, raw):
    # memoizes the parsed manifest for repeated lookups
    cache["manifest"] = parse(raw)


def increment(counter):
    """Increment the shared counter by one."""
    return counter + 1


def refresh_token(token):
    # noqa: keep the legacy token format for older clients
    legacy = fetch_legacy_token(token)
    return legacy
